### PR TITLE
Nc news ticket#7

### PR DESCRIPTION
**Pull request by reece-b4**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -77,24 +77,68 @@ describe("/api/articles/:article_id", () => {
...
+    test(" 400- given incorrect votes data type returns message: bad request", () => {
+      const votes = { inc_votes: 'not-a-valid-vote-count' };
+      return request(app)
+        .patch("/api/articles/999999")
+        .send(votes)
+        .expect(400)
+        .then((res) => {
+          expect(res.body.msg).toBe("bad request");
+        });
+    });
```
Comment (cjplatten) [inline __tests__/app.test.js:150]: what happens if the patch request is sent with the wrong key or an empty object?

**Review by cjplatten — COMMENTED** [review]
See comment

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+      const votes = { inc_votes: -10 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(votes)
+        .expect(200)
+        .then((res) => {
+          expect(res.body.article).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
```
Comment (popatre) [inline __tests__/app.test.js:93]: Should be an easy one - but have you tested if it actually increments the votes?
At the moment, all the tests check that is decrements.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+        });
+    });
+    test(" 400- given empty object returns message: bad request", () => {
+      const votes = {};
+      return request(app)
+        .patch("/api/articles/1")
+        .send(votes)
+        .expect(400)
+        .then((res) => {
+          expect(res.body.msg).toBe("bad request");
```
Comment (popatre) [inline __tests__/app.test.js:186]: Test - what if there are more than one key on the object? e.g. inc_votes and title. 

It should be handled already, but good to show its been considered.

**Review by popatre — CHANGES_REQUESTED** [review]

File: __tests__/app.test.js (hunk 2 above)
Comment (reece-b4) [reply __tests__/app.test.js:93]: I have tested yeah I actually just changed the test but can add it in easily

**Review by Augs0 — APPROVED** [review]
Just to note that you only need one 404 test globally - no need to use this on every endpoint

**Pull request merged**